test/adapter/json_api/relationship_test.rb

```diff
@@ -171,6 +397,42 @@ def test_relationship(expected, test_options = {})
           relationship = Relationship.new(parent_serializer, serializable_resource_options, association)
           assert_equal(expected, relationship.as_json)
         end
+
+        def build_serializer_and_serialize_relationship(model, relationship_name, &block)
+          serializer_class = Class.new(ActiveModel::Serializer, &block)
+          hash = serializable(model, serializer: serializer_class, adapter: :json_api).serializable_hash
+          hash[:data][:relationships][relationship_name]
+        end
+
+        def new_model(model_attributes)
```

Comment:
specifically the dynamic model and serializer are just to get around gotchas from the 'poro fixtures', like caching or any other quirks.